Modules/BenchPress.Azure/Tests/Public/Confirm-OperationalInsightsWorkspace.Tests.ps1

```diff
@@ -8,10 +8,10 @@ Describe "Confirm-OperationalInsightsWorkspace" {
   Context "unit tests" -Tag "Unit" {
     BeforeEach {
       Mock Connect-Account{}
+      Mock Get-AzOperationalInsightsWorkspace{}
```

Comment:
Is there a reason we are mocking these `get-az` calls? Thought all we needed was to check it is called